### ISSUE TITLE
getDeviceUUID doesn't work for ZFS

### ISSUE THREAD

**Issue by Random-Liu** [kubernetes-incubator/cri-containerd#325]
Mount info for ZFS is like this:
```
tmpfs on /var type tmpfs (rw,nosuid,nodev,noexec,relatime,size=1023524k,mode=755)
testpool/nextnas/containerd-vol on /var/lib/containerd/io.containerd.snapshotter.v1.zfs type zfs (rw,xattr,noacl)
testpool on /var/zpool type zfs (rw,xattr,noacl)
testpool/nextnas on /var/zpool/nextnas type zfs (rw,xattr,noacl)
```

The pool is not a file on the disk, thus could not be stated.

We need to figure out how to get device UUID for zfs. Or maybe we need admin to pass the UUID to us for zfs if we could not figure it out.

**Comment by Random-Liu** [comment]
Reopen this issue. According to https://github.com/kubernetes-incubator/cri-containerd/pull/335#issuecomment-336724360, ZFS still doesn't work.

The main problem is that ZFS doesn't have an uuid, because it manages the block/volume itself.

I feel like we should throw out an error and proceeding if we could not get uuid, the only thing won't work is that kubelet can't get image fs capacity. However, we should not block people from trying this out because of that.

**Comment by Random-Liu** [comment]
We added `skip-imagefs-uuid` flag, and will be included in `v1.0.0-beta.1`. If the current uuid based approach doesn't work, user could use that flag to skip that step. It should not prevent user from trying out this.

I'll close this issue. Further discussion will be carried on in the kubernetes issue kubernetes/kubernetes#57356.
